SwaprHQ/swapr-dapp
# [SWA-146][FEAT] - Add Openocean support

## Fixes: [SWA-146](https://linear.app/swaprhq/issue/SWA-146/add-openocean-support)

# Description
* Add Kleros as a routing platform

# Visual Evidence
![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/e87a73bf-fd20-4e3c-b3bb-654e97f518f1)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Connect your wallet
5) Go to the Swapbox, select the pair you want to swap on any of the supported chains by Openocean (Arbitrum, BSC, Gnosis, Ethereum, Optimism, Polygon, Scroll) and Openocean should appear as a valid route.
